lib/compress.js

```diff
@@ -1731,6 +1731,7 @@ merge(Compressor.prototype, {
                 var val = {};
                 for (var i = 0, len = this.properties.length; i < len; i++) {
                     var prop = this.properties[i];
+                    if (prop instanceof AST_Expansion) return this;
```

Comment:
I see you've started working on `compress` support :paperclip:

Scanning through `.properties` in `lib/compress.js`, I see the following potential changes:
- `may_throw_on_access()` need to `return true` on `AST_Object` with `AST_Expansion`
- `evaluate()` will need to bail out on `unsafe` of `AST_Object` with `AST_Expansion`
- `drop_side_effect_free()` will need to extract `AST_Expansion.value` when flattening `AST_Object`